src/index.js

```diff
@@ -46,7 +56,28 @@ const onLoad = (userID) => {
       })
       activeUser.sortBookings();
       assignGlobals(roomsRepo, bookingsRepo, activeUser);
-      updateUserBookings();
+      populateUserAside(activeUser);
+    })
+    .catch(err => console.log(err))
+}
+
+const managerLoad = () => {
+  Promise.all([getRooms(), getBookings(), getAllUsers()])
+    .then(([loadedRooms, loadedBookings, loadedUsers]) => {
+      const roomsRepo = new Rooms (loadedRooms.rooms);
+      const bookingsRepo = new Bookings (loadedBookings.bookings);
+      const usersRepo = [];
+      loadedUsers.customers.forEach(user => {
+        const newU = new User (user)
+        usersRepo.push(newU)
+      })
+      usersRepo.forEach(user => {
+        bookingsRepo.bookingsByUser(user.id).forEach(entry => {
+          return user.addToBookingsRecord(entry);
+        })
+        user.sortBookings();
+      })
+      managerGlobals(roomsRepo, bookingsRepo, usersRepo)
```

Comment:
I didn't do this extension so I might be missing something, but I wonder if its possible to consolidate this function with your customer load function instead of fetching the data for rooms and bookings twice.  It would probably end up being a significant refactor, but there might be a good trade off when it comes to cutting back on async operations.

Reply:
Consolidating them might leave you with fetching more than what you really need (like a single customer who is logged in doesn't require the allCustomers data), but perhaps a conditional could prevent that fetch request.